packages/svelte/src/internal/client/dom/elements/events.js

```diff
@@ -66,6 +66,20 @@ export function create_event(event_name, dom, handler, options) {
 	return target_handler;
 }
 
+/**
+ * @param {Element} dom
```

Comment:
A bit of inline documentation might be nice (why you would need this over the regular addEventListener)